Chatto/Source/ChatController/BaseChatViewController+Scrolling.swift

```diff
@@ -36,79 +36,97 @@ extension CGFloat {
 extension BaseChatViewController {
 
     public func isScrolledAtBottom() -> Bool {
-        guard self.collectionView.numberOfSections > 0 && self.collectionView.numberOfItems(inSection: 0) > 0 else { return true }
-        let sectionIndex = self.collectionView.numberOfSections - 1
-        let itemIndex = self.collectionView.numberOfItems(inSection: sectionIndex) - 1
+        guard
```

Comment:
Could I ask you to use several guard statements here and below ? It would be more readable:

```
guard let collectionView = self.collectionView else { return true }
guard let ... else { return true }
```

Reply:
I think it easier to read when you have only return statement, but OK